packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -130,6 +143,22 @@ const VideoEffectsPaneTrampoline = (
   selectableVideoEffects?: _VideoEffectsItemProps[],
   onEffectChange?: (effectKey: string) => Promise<void>
 ): JSX.Element => {
+  /* @conditional-compile-remove(video-background-effects) */
+  const [dismissedError, setDismissedError] = useState<DismissedError>();
+  /* @conditional-compile-remove(video-background-effects) */
+  const latestEffectError = useSelector(videoBackgroundErrorsSelector);
+  /* @conditional-compile-remove(video-background-effects) */
+  let showError = false;
+  /* @conditional-compile-remove(video-background-effects) */
+  if (latestEffectError) {
+    if (dismissedError) {
+      showError = latestEffectError?.timestamp > dismissedError.dismissedAt;
+    } else {
+      showError = true;
+    }
+  }
+  /* @conditional-compile-remove(video-background-effects) */
+  useEffect(() => setDismissedError(dismissedError), [dismissedError]);
```

Comment:
what is the purpose of this useEffect? It looks like any time `dismissedError` changes we call `setDismissedError` (which feels like a no-op as it will set the dimissedError to the value it already is)